pkg/library/flatten/workspaceEnv.go

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright (c) 2019-2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package flatten
+
+import (
+	"fmt"
+
+	dw "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+const (
+	// WorkspaceEnvAttribute is an attribute that specifies a set of environment variables provided by a component
+	// that should be added to all workspace containers. The structure of the attribute value should be a map of strings
```

Comment:
> The structure of the attribute value should be a map of strings
> 	// to strings, e.g.

it's not true, right? Because it's an array of EnvVar objects

Reply:
Yup, forgot to update the comment as a switched back and forth between the structures :)